packages/fluent2-theme/src/componentStyles/SpinButton.styles.ts

```diff
@@ -1,20 +1,32 @@
 import type { ISpinButtonStyleProps, ISpinButtonStyles, IStyleFunctionOrObject } from '@fluentui/react';
-import { FontWeights, getInputFocusStyle } from '@fluentui/react';
+import { FontWeights } from '@fluentui/react';
 
 export function getSpinButtonStyles(
   props: ISpinButtonStyleProps,
 ): IStyleFunctionOrObject<ISpinButtonStyleProps, ISpinButtonStyles> {
   const { theme, isFocused, disabled } = props;
   const { semanticColors } = theme;
-  const SpinButtonRootBorderColor = semanticColors.inputBorder;
-  const SpinButtonRootBorderColorHovered = semanticColors.inputBorderHovered;
-  const SpinButtonRootBorderColorFocused = semanticColors.inputFocusBorderAlt;
+  const SpinButtonRootBorderColorFocused = semanticColors.disabledBorder; // sorry for the broken semantics
```

Comment:
nit: Might be helpful to include why we're not lining up here.

Reply:
Mainly so if we need to change it's able to be maintained even if you aren't the one making the changes down the road.